.env.development

```diff
@@ -0,0 +1 @@
+GATSBY_SCREENSHOT_PLACEHOLDER=true
```

Comment:
You’ll want to remove this file (or I can!) from this PR.

Also just a note — this should “live” in www, e.g. www/.env.development!